internal/cleanup/cleanup.go

```diff
@@ -127,7 +129,7 @@ func (h *exportCleanupHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	timeoutCtx, cancel := context.WithTimeout(ctx, h.config.Timeout)
 	defer cancel()
 
-	count, err := h.database.DeleteFilesBefore(timeoutCtx, cutoff, h.blobstore)
+	count, err := exportdb.NewExportDB(h.database).DeleteFilesBefore(timeoutCtx, cutoff, h.blobstore)
```

Comment:
I don't know if we need these to be typed or just package level methods.
Having worked briefly w/ the first refactor I think I would prefer

```exportdb.DelteFiledBefore(h.databse, ...)```

Reply:
@mikehelmick I am pretty indifferent and could make this change. I had thought perhaps once fully moved and separated the handler could be updates do `h.database` was pointing to `exportdb` (so you wouldn't be making the objects) 

Something like:
```
return &exposureCleanupHandler{
		config:   config,
		env:      env,
		exposureDatabase: exposuredb.NewExposureDB(env.Database()),
		exportDatabase: exportdb.NewExportDB(env.Database()),
	}, nil
```

Reply:
Is `h.database` a reference to the core DB object? So anyone with that could execute arbitrary statements against the DB? I thought part of the idea was to prevent the api packages (like `internal/cleanup`) from holding on to one of those.

Reply:
correct `h.database` is a `coredb` object reference. `env.Database()` provides one. By moving it into the `exposureCleanupHandler`, it would be hard to accidentally be using it though. Also, as part of this refactor `h.database` has none of the functions of tables, but has functions like `Pool`, `InTx`

Reply:
Got it. So then, I like your design two comments up where the handler struct holds only the wrapped DB handles. Then it is clear from inspection that the handler methods can't do anything with core handle; they have to use the api-specific ones.

Reply:
Thanks for the feedback. I have made the changes and pushed.